config/initializers/devise.rb

```diff
@@ -0,0 +1,277 @@
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = 'ec140ef1f599048203d1e8c90b7cdf32336a02975d1c27fd98f306138a7505a1aacb68fb8998c64e96aebf1a47e7774a59781a89424f1ad0af9d52668a2ea4e3'
+
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer,
+  # note that it will be overwritten if you use your own mailer class
+  # with default "from" parameter.
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```

Comment:
Lets use the secrets file to store the mail sender. Check [this post](https://richonrails.com/articles/the-rails-4-1-secrets-yml-file) (ignore the part where suggests add the secrets.yml file to the .gitignore)